README.md

```diff
@@ -3,40 +3,68 @@ The repo for the terra data repository built by the jade team.
 
 See the DATABASE.md to set up the postgres database before you run jade.
 
-## Setup environment
-### Google project and account
+## Create kubernetes cluster
+
+In the google cloud console, within your personal project, go to Kubernetes Engine -> Clusters and create a cluster. In the default pool section change nodes to 1 and machine type to small. Open the Advanced section and click the checkbox to enable VPC native. Then click create.
+
+Once your cluster has finished creating, click the Connect button next to you cluster info. Copy the command and execute it on your local system. Now, if you click on docker -> kubernetes you should see a check next to the cluster you just created.
+
+
+## Create a service account
+In the google cloud console, go to IAM & Admin
+
+## Deploying to kubernetes
+### Deploying in your own test account (not dev, integration, etc)
+#### Environment variables
+    GOOGLE_CLOUD_PROJECT
+    ENVIRONMENT (local, dev)
+    
+    
+Add the service account key for your project to vault:
+
+    vault write secret/dsde/firecloud/local/datarepo/sa-key{project}.json @<localfilename>.json
```

Comment:
There is probably something I need to read before this that would tell me to install vault on my system and set it up some way?

Reply:
i'd be surprised if you don't already have it. i'm not sure exactly where i got it.